.github/ISSUE_TEMPLATE/ovms-bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: OVMS Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Steps to prepare models repository '...'
+2. OVMS launch command '....'
+3. Client command '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Logs from OVMS, ideally with --log_level DEBUG. Logs from client.
+
+**Configuration files**
```

Comment:
Some machine and inference device details might be useful?
Docker image used (but I suppose we would get that from OVMS launch command)?

Reply:
I added request for HW. Yes we should get the docker image from the launch command. I want to keep this list short and simple.